server/functions/getMyTeamMembers/source.js

```diff
@@ -2,10 +2,11 @@ exports = async function() {
   const cluster = context.services.get("mongodb-atlas");
   const collection = cluster.db("tracker").collection("User");
   const caller = context.user;
+  const projectPartition = `project=${caller.id}`;
   const filter = {
     $or: [
-      {"readable_partitions": caller.id}, // has my id as a readable partition
-      {"writeable_partitions": caller.id}, // has my id as a writeable partition
+      {"canReadPartitions": projectPartition}, // has my project id as a readable partition or
+      {"canWritePartitions": projectPartition}, // has my project id as a writeable partition
```

Comment:
I actually prefer the terms "readable/writable". While I don't think the new terms are *bad*, is there some reason for the change? Or just personal preference?